datastore/googlepubsub/publisher.go

```diff
@@ -143,3 +143,8 @@ func registerMetrics(metricsCollector metrics.MetricCollector) {
 		Labels: []string{"record_type"},
 	})
 }
+
+// Close the underlying pubsub connection.
+func (p *Producer) Close() error {
+	return p.pubsubClient.Close()
```

Comment:
should we also stop publishing topics?
```sh
	pubsubTopic.Stop()
```
Right now we don't store this as an object as producer level. Maybe we should persist and close them?